exercise-book/src/simple-db.md

```diff
@@ -62,7 +62,7 @@ With the additional properties:
 4. Empty payloads are allowed. In this case, the command is
     `PUBLISH \n`.
 
-Violations against the form of the messages and the properties are
+Errors in the form or properties of the messages are
```

Comment:
I think this got mangled?

Reply:
I was intentionally writing around violations, but we can either rewrite or igore.

Reply:
```suggestion
Issues with the format (or other properties) of the messages are
```

perhaps?